DockerFileTests

```diff
@@ -18,8 +18,13 @@ RUN mkdir -p /usr/local/gcloud \
 
 FROM tests_base AS with_requirements
 
-COPY tests/requirements.txt .
-RUN pip install -r requirements.txt
+# Changing the installation of packages
+# COPY tests/requirements.txt .
+# RUN pip install -r requirements.txt
+RUN pip install pipenv
+COPY tests/Pipfile .
+COPY tests/Pipfile.lock .
+RUN pipenv install --system --ignore-pipfile
```

Comment:
Why `--ignore-pipfile`?

Reply:
The `--ignore-pipfile` downloads all the dependencies from the Pipfile.lock, and I read that this is a better way to ensure deterministic builds.

Reply:
Good to know! There's no build going on here, and it's not determinstic as it's just a test, so I think it's okay to skip this.

Reply:
Fixed this!